README.md

```diff
@@ -494,6 +495,43 @@ jobs:
           password: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
 ```
 
+### Buildkite Package Registry
+
+Use your Buildkite registered email address to generate an API access token to authenticate. U
```

Comment:
Looks like a typo here:

```suggestion
Use your Buildkite registered email address to generate an API access token to authenticate.
```

Also this looks like a copy/paste of DigitalOcean example and does not seem accurate looking at Buildkite docs. I think the following would be better to be consistent with other examples related to my previous comment:

```suggestion
Configure [API access token](https://buildkite.com/user/api-access-tokens) to login to [Buildkite Package Registries](https://buildkite.com/docs/package-registries/container). Ensure this access token has the **Read Packages** and **Write Packages** REST API scopes.
```